pkg/securelaunch/helpers.go

```diff
@@ -0,0 +1,291 @@
+// Copyright 2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package securelaunch
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"github.com/u-root/u-root/pkg/cmdline"
+	"github.com/u-root/u-root/pkg/diskboot"
+	"github.com/u-root/u-root/pkg/find"
+	"github.com/u-root/u-root/pkg/mount"
+	"github.com/u-root/u-root/pkg/storage"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+var storageBlkDevices []storage.BlockDev
+var Debug = func(string, ...interface{}) {}
+
+/*
+ * modified ver of https://stackoverflow.com/questions/10510691/how-to-check-whether-a-file-or-directory-exists
+ * exists returns whether the given file or directory exists
+ * also returns if the path is a directory
+ */
+func exists(path string) (bool, bool, error) {
+	fileInfo, err := os.Stat(path)
+	if err == nil {
+		return true, fileInfo.IsDir(), nil
+	}
+	if os.IsNotExist(err) {
+		return false, false, nil
+	}
+	// file or dir def exists..
+	return true, fileInfo.IsDir(), err
```

Comment:
is there any issue if this is a symlink, socket, special, or other? IOW do you want to be the return 
return fileinfo.IsRegular(), fileinfot.IsDir(), error?

Or maybe error is set if it's not file or dir, and the first return is which it is?

Reply:
> is there any issue if this is a symlink, socket, special, or other? 
Nope, doesnt matter.. Its a JSON file that user sets up. We really dont care what kind of file it is.